metals/src/main/scala/scala/meta/internal/metals/codeactions/StringActions.scala

```diff
@@ -0,0 +1,97 @@
+package scala.meta.internal.metals.codeactions
+
+import scala.meta.internal.metals.CodeAction
+import scala.meta.internal.metals.Buffers
+import scala.meta.internal.metals.MetalsEnrichments._
+import org.eclipse.{lsp4j => l}
+import scala.concurrent.{ExecutionContext, Future}
+import scala.meta.pc.CancelToken
+import scala.meta.tokens.Token
+
+class StringActions(buffers: Buffers) extends CodeAction {
+
+  override def kind: String = l.CodeActionKind.Refactor
+
+  override def contribute(
+      params: l.CodeActionParams,
+      token: CancelToken
+  )(implicit ec: ExecutionContext): Future[Seq[l.CodeAction]] = {
+
+    val uri = params.getTextDocument.getUri
+    val path = uri.toAbsolutePath
+    val range = params.getRange
+
+    Future
+      .successful {
+        path.toInputFromBuffers(buffers).tokenize.toOption match {
+          case Some(tokens) =>
+            val stringActions = tokens.collect {
+              case token: Token.Constant.String
+                  if (token.pos.toLSP.overlapsWith(range) && isNotTripleQuote(
+                    token
+                  )) =>
+                stripMarginAction(uri, token.pos.toLSP)
```

Comment:
```suggestion
token
```
and then we can handle it together with interpolated strings, so no need to iterate over the tokens twice.